frontend/src/features/admin-form/responses/ResponsesPage/storage/worker/decryption.worker.ts

```diff
@@ -227,3 +257,8 @@ const exports = {
 expose(exports)
 
 export type DecryptionWorkerApi = typeof exports
+
+//TODO: move this to a util file?
```

Comment:
Yes 😆.

Can extract this out into `shared/utils/is-string-array.ts` and have `/Users/kenlee/Project/formSG3/src/app/utils/field-validation/field-validation.guards.ts` import it as well.